### PR TITLE
Prettier and CaptainCommand

### PR DESCRIPTION
**Pull request by Luke-Bytes**
/captain set [discord] [team]

-Must be registered player
-Adds you to the team colour and sets isCaptain true
-Removes previous captain if they exist

### CODE REVIEW

File: src/commands/RoleCommand.ts
Comment (99-i) [inline src/commands/RoleCommand.ts:1]: should this file be deleted?

**Review by 99-i — CHANGES_REQUESTED** [review]
looked fine, but RoleCommand.ts contains another implementation of CaptainCommand?

File: src/commands/CaptainCommand.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    player: PlayerData,
+    teamColor: "blue" | "red"
+  ): boolean {
+    const inGameName = player.getInGameName();
+    if (teamColor === "blue") {
+      return GameData.getBluePlayers().includes(inGameName);
+    } else {
+      return GameData.getRedPlayers().includes(inGameName);
+    }
+  }
```
[the remark this answers is not in the document]
Comment (Luke-Bytes) [reply src/commands/CaptainCommand.ts:201]: Let's add to Utils, it probably can be reused later

File: src/commands/RoleCommand.ts
Comment (Luke-Bytes) [reply src/commands/RoleCommand.ts:1]: Reverted now

File: src/commands/CaptainCommand.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          console.error(`Failed to add captain role: ${error}`)
+        );
+
+      if (teamColor === "blue") {
+        GameData.setBluePlayers([...GameData.getBluePlayers(), inGameName]);
+        console.log(`${inGameName} added to bluePlayers`);
+      } else {
+        GameData.setRedPlayers([...GameData.getRedPlayers(), inGameName]);
+        console.log(`${inGameName} added to redPlayers`);
+      }
```
[the remark this answers is not in the document]
Comment (Luke-Bytes) [reply src/commands/CaptainCommand.ts:158]: In reality I think this won't happen as teams won't be assigned until a definitive captain has started picking teams but it's still an edge case we can fix I'll add something for that

[the remark this answers is not in the document]
Comment (Luke-Bytes) [reply src/commands/CaptainCommand.ts]: I realised if you wanted to change the colour of the captain it borked stuff, I think every edge case is handled now (switching  captains between players and teams related)

**Review by 99-i — APPROVED** [review]

**Pull request merged**